### ISSUE TITLE
Paused show hides episodes manually set to wanted

### ISSUE THREAD

**Issue by nsgundy** [SiCKRAGETV/sickrage-issues#1138]
Had a show that was set to `paused`. Seasons 1 and 2 had been previously `downloaded`, seasons 3, 4, 5 and 6 were set to `skipped`. When I marked seasons 3 and 4 and set the status to `wanted`, seasons 3 and 4 disappeared from the show page entirely (the top-right season anchors were still there though). SickRage didn't try to fetch these episodes either, it was just like they vanished.
Only after removing the `paused` state did seasons 3 and 4 reappear. Their status was `wanted` but SickRage did not start looking for them until I manually set them to `wanted` once more (possibly forcing backlog processing would have done the job).

Besides the shows not disappearing, I would have expected that even though the show was set to `paused`, SickRage would have snatched seasons 3 and 4 after I had manually set them to `wanted`.

**Comment by Thraxis** [comment]
Pausing a show tells SickRage you want to ignore it for now.  So no it would not download anything marked as wanted or even any new episodes that come up.

**Comment by flightlevel** [comment]
From a logic point of view, you'd expect that when a show is paused it would stop from downloading new episodes, but if you make an episode wanted it would download. ie. pause stops automatic things from happening, but it shouldn't block when the user has intervened. Why would you make something wanted if you didn't want it?

A common situation is, you decide to start watching a series that is currently upto season 8. In the past I would have added the show as paused, as I don't want SickRage to be downloading season 8 episode 14 etc, when I still haven't watched the first few episodes of season 1 to see if I like that show and will continue watching it.

**Comment by fernandog** [comment]
changed to feature request

**Comment by nsgundy** [comment]
This report has two elements:
* Episodes marked as `wanted` don't show up in the list at all for a `paused` show. I would think this is a bug.
* SickRage to fetch episodes manually set to `wanted` on a `paused` show, exactly for the reasons given by @flightlevel.

**Comment by fernandog** [comment]
@nsgundy so make another issue for the the possible bug with screenshots, logs and everything. Keep this as FR to force download when user manually set as wanted and show is paused 

 but this FR will conflict with
https://github.com/SiCKRAGETV/sickrage-issues/issues/878

@djack42

**Comment by abeloin** [comment]
IMHO the FR should be closed with won't fix. If we allow the WANTED status to bypass the paused status, we need to change the search as it skip the paused show without checking for wanted episode, trakt rolling isn't enable when the show is paused. All new episode are set to skipped on paused, do we change this? Also some users will complaint the it didn't download the best quality or refresh to best quality because search daily ignore paused show.

This is plain wrong, the reason for paused is to prevent any download which the OP want to bypass. He should set the pause status to false and if he doesn't to download new episodes, set the default status for that show to Ignored

**Comment by nsgundy** [comment]
Sorry @fernandog, this was mainly the bug report (see the description), the "FR" was just thrown in there as something I noticed being different from what I thought should be going on and it is just a foot note (see initial post), however everyone seemed to have ignored that and put all focus on the "FR".

It is super reproducible. Just spotted it on one of my other paused shows, so here are some screen shots.
Missing shows when paused: ![http://i.imgur.com/b4NpmVWm.png](http://i.imgur.com/b4NpmVW.png)
Reappearing after un-pausing: ![http://i.imgur.com/voQbOZAm.png](http://i.imgur.com/voQbOZA.png)
Running BRANCH: (master) / COMMIT: (669046ea40a7a26a6121a1b07721f3c1c3cf5b10)

**Comment by fernandog** [comment]
@abeloin javascript issue?

**Comment by abeloin** [comment]
@nsgundy @fernandog PR merged in develop fixing the issue
